.gitignore

```diff
@@ -0,0 +1,2 @@
+.stack-work
+*.hie
```

Comment:
Not harmful, but I would never expect `.hie` files to land at top-level. I'm curious if/how you find them.

Reply:
You're right, I had to try that out.

I could've been more precise though,

Not harmful, but I would never expect .hie files to land ~~at top-level~~ outside of the already-ignored `.stack-work` directory. I'm curious if/how you find them.